deployments/cluster/witness-load-balancer.nix

```diff
@@ -1,27 +1,49 @@
 # TODO: add AWS ALB to NixOps and use that instead
 
-domain: { config, pkgs, ... }:
+env: domain: { config, lib, pkgs, resources, ... }:
 
+let
+  uris = {
+    faucet = "faucet.${domain}";
+    explorer = "explorer.${domain}";
+    educator = "educator.${domain}";
+    witness = "witness.${domain}";
+  };
+
+in
 {
-  deployment.route53.hostName = "witness.${domain}";
+  deployment.route53.hostName = lib.mkForce "witness.${domain}";
+
+  deployment.ec2.securityGroupIds = map (x: resources.ec2SecurityGroups."cluster-${x}-sg".name ) (
+    [ "http-public" ]
+  );
+
+  deployment.keys."faucet-key.json".keyFile = ../../keys + "/${env}/faucet-key.json";
```

Comment:
this is the only place `env` is used. maybe this should be somewhere else?

Reply:
We use it to load per-environment secrets. You did something about that, but it was deleted along with the rest.